core/src/main/java/com/rudderstack/core/Logger.kt

```diff
@@ -45,11 +45,11 @@ interface Logger {
      * @property level priority for each type
      */
     enum class LogLevel(internal val level: Int){
-        Info(0),
-        Debug(1),
-        Warn(2),
-        Error(3),
-        None(4),
+        DEBUG(0),
+        INFO(1),
+        WARN(2),
+        ERROR(3),
+        NONE(4),
```

Comment:
Can we cross-test if we actually need these `level: Int values`?
These are utilised by `KotlinLogger.kt`, so I was wondering if turning the `KotlinLogger.kt` to this would make the trick:
```


class KotlinLogger(level: Logger.LogLevel) : Logger {
    private var logLevel: Logger.LogLevel = level
        @Synchronized set
        @Synchronized get

    override fun activate(level: Logger.LogLevel) {
        logLevel = level
    }

    override fun info(tag: String, log: String) {
        if (Logger.LogLevel.INFO >= logLevel)
            println("$tag-info : $log")
    }

    override fun debug(tag: String, log: String) {
        if (Logger.LogLevel.DEBUG >= logLevel)
            println("$tag-debug : $log")
    }

    override fun warn(tag: String, log: String) {
        if (Logger.LogLevel.WARN >= logLevel)
            println("$tag-warn : $log")
    }

    override fun error(tag: String, log: String, throwable: Throwable?) {
        if (Logger.LogLevel.ERROR >= logLevel)
            println("$tag-error : $log")
    }

    override val level: Logger.LogLevel
        get() = logLevel
}

```

Reply:
I just checked with @itsdebs, and this `level: Int values` is not required. I'll also remove it and make the suggested changes in the `KotlinLogger` class. Thanks a lot for the suggestion.